examples/auctions/blind_auction.vy

```diff
@@ -0,0 +1,63 @@
+# Blind Auction
+# Adapted to Vyper from [Solidity by Example](https://github.com/ethereum/solidity/blob/develop/docs/solidity-by-example.rst#blind-auction-1)
+
+# Custom struct for bids
+bid: {
+    blindedBid: bytes32,
+    deposit: wei_value
+}
+
+# Auction parameters
+beneficiary: public(address)
+biddingEnd: public(timestamp)
+revealEnd: public(timestamp)
+
+# Set to true at the end of auction, disallowing any new bids
+ended: public(bool)
+
+# State of bids
+bids: public([address])
+
+# Final auction state
+highestBidder: public(bid[address])
```

Comment:
As noted above `bid` is not a named struct, so this will not work.

I would also sort of expect this to just be `highestBidder: address`, which can be used to look up the owner of the current highest bid (which probably doesn't mean much after revealing, you could probably delete it). It seems like you are doing this after a commit-reveal period, I would add some comments to that effect to differentiate globals from stage 1 (commit) and stage 2 (reveal).